src/views/splash/presentation.jsx

```diff
@@ -31,6 +31,10 @@ const LoveProjectMessage = require('./activity-rows/love-project.jsx');
 const RemixProjectMessage = require('./activity-rows/remix-project.jsx');
 const ShareProjectMessage = require('./activity-rows/share-project.jsx');
 
+// Beta Banner Components
+const TopBanner = require('./beta/top-banner.jsx').default;
+const MiddleBanner = require('./beta/middle-banner.jsx').default;
```

Comment:
After using `module.exports`, this can be updated to not have to use `.default`.